msp/msp_opcodes.h

```diff
@@ -43,6 +43,8 @@
 #define MSP_OP_SEND_CUBES_HVPS_TMP_VOLT			0x75
 #define MSP_OP_SEND_READ_REG_DEBUG				0x76
 #define MSP_OP_SEND_CUBES_GATEWARE_CONF			0x77
+#define MSP_OP_SEND_NVM_CITI_CONF               0x78
+#define MSP_OP_SELECT_NVM_CITI_CONF             0x79
```

Comment:
These need to be redefined as was done on `proto-cubes-obc`. (The Project 2 PR which was already merged already defined `0x78`.)